docs/modules/ROOT/pages/partials/con_hermetic_verification.adoc

```diff
@@ -1,16 +1,16 @@
 .Verification
 
-. On your application, go to the *Activity > Pipeline runs* tab.
+. From the *Applications* view, go to *Activity > Pipeline runs*.
```

Comment:
They could also see this from the commit tab, and look for whatever PR they just created.

Reply:
ahhh. i didn't touch this file but i can certainly make this change!